jekyll/_cci2/triggers-overview.adoc

```diff
@@ -63,6 +65,35 @@ curl -X POST https://circleci.com/api/v2/project/<VCS>/<your-vcs-org-or-username
 . Head back to the CircleCI web app and see your pipeline running on the dashboard.
 
 TIP: You can also specify pipeline parameters when triggering a pipeline using the API. For full details, see the https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline[API v2 Docs].
+--
+
+[.tab.apitrigger.GitHub_App_/_Bitbucket_Data_Center]
+--
+For **projects integrated with GitHub App or Bitbucket Data Center**, you can trigger a pipeline for a project using the endpoint `https://circleci.com/api/v2/project/{project-slug}/pipeline/run`.
+
+NOTE: This endpoint is currently in preview, and is not yet documented in the link:https://circleci.com/docs/api/v2/index.html[official API V2 documentation].
+
+. If you have not already, get set up to use API v2 by following the steps in the  <<api-developers-guide#authentication-and-authorization,API Developers Guide>>. Here you will find instructions on how to find your project slug.
```

Comment:
I think it would be better to explain project-slug here rather than leaving people to find it on the other page, is that ok if I add that?

Reply:
Of course! Thank you